docs/maatregelen/2-owp-20-vaststellen-aanleveren-informatie-technische-documentatie.md

```diff
@@ -0,0 +1,54 @@
+---
+# vul hier een titel in voor deze maatregel
+title: Vul technische documentatie van aanbieder aan met relevante informatie vanuit de gebruiksverantwoordelijke. 
+id: urn:nl:ak:mtr:owp-20
+# geef hier een korte toelichting van deze maatregel
+toelichting: Bespreek met het projectteam welke onderdelen van de technische documentatie, als genoemd in de Bijlage 4 AI-verordening, van het algoritme of AI-systeem door welke partij (aanbieder of gebruiksverantwoordelijke) moeten worden ingevuld of aangevuld. 
```

Comment:
```suggestion
toelichting: Bespreek met het projectteam welke onderdelen van de technische documentatie (zoals genoemd in de AI-verordening) van het algoritme door wie moeten worden ingevuld. 
```

De toelichting wordt soms in tabellen gebruikt volgens mij, dus suggestie gedaan om deze iets korter te maken ivm leesbaarheid.